From 2d0c3e3812f72bd0f8c0d018ad525494b631c702 Mon Sep 17 00:00:00 2001
From: Albin B <albin.bala.krishnan@intel.com>
Date: Tue, 2 Feb 2016 15:52:14 +0530
Subject: [PATCH 10/13] EM/bq24192_charger: Fix for usb compliance
 suspend/resume issue

When the device is compliant with usb, device should disable the
charging when host goes to suspend. Device is not disabling the
charging/charger even when the otg notifies the inlimit current
2mA and  pmic driver is not updating the same. Hence it doesn't
disable the charger and charging. Now allows to set the inlimit
current based on the otg event to make the usb compliant.

Change-Id: Idc25cc9b695bbeef0bb836b1945f0b495c9dabfc
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-17204
Signed-off-by: Albin B <albin.bala.krishnan@intel.com>
Reviewed-on: https://android.intel.com:443/465417
---
 drivers/power/bq24192_charger.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/power/bq24192_charger.c b/drivers/power/bq24192_charger.c
index 8d08325..d12a026 100644
--- a/drivers/power/bq24192_charger.c
+++ b/drivers/power/bq24192_charger.c
@@ -238,7 +238,7 @@
 #define BQ_GPIO_MUX_SEL_PMIC	0
 #define BQ_GPIO_MUX_SEL_SOC	1
 #define FPO0_USB_COMP_OFFSET	0x01
-#define BQ24192_MIN_INLMT	100
+#define BQ24192_MIN_INLMT	0
 #define BQ24192_MAX_INLMT	3000
 #define BQ24192_MAX_ITERM	2048
 static struct power_supply *fg_psy;
-- 
1.9.1

